src/api/tests/test_revision_pull.py

```diff
@@ -21,12 +21,17 @@ def test_get_records(db_setup, project1, project2, project3, profile1, project_p
 
     collect_record.save()
     CollectRecord.objects.create(project=project1, profile=profile1, data=dict())
+    for rev in Revision.objects.all():
+        print(f"[{rev.revision_num}] {rev.table_name} {rev.record_id} {rev.deleted} {rev.related_to_profile_id}")
 
-    updates, deletes = get_records(
+    updates, deletes, removes = get_records(
         collect_record_viewset,
-        None,
-        collect_record.project_id,
-        collect_record.profile_id,
+        profile1.pk,
+        required_params={
+            "revision_num": None,
+            "project_id": collect_record.project_id,
+            "profile_id": collect_record.profile_id,
```

Comment:
Shouldn't the `required_params` keys be `project` and `profile`? (Maybe naming should be made consistent?)